process/project_proposals.adoc

```diff
@@ -69,7 +63,7 @@ image::incubation-process.png[Incubation process]
    * Project fills out and submits the link:graduation-proposal-template.md[graduation proposal template] in a pull request in the https://github.com/cncf/toc[cncf/toc GitHub repo].
    * The file containing the proposal should be located in https://github.com/cncf/toc/tree/main/proposals/graduation[the graduation proposals directory].
    * The proposal addresses how the project has grown since incubation and any concerns from incubation DD in addition to the standard graduation requirements.
-. *TOC member kicks off two week period of time for pulic comment on the TOC mailing list*
+. *TOC member kicks off two week period of time for public comment on the TOC mailing list*
    * The email should contain a link to the proposal pull request.
```

Comment:
"The email should contain a link to the proposal pull request" ...and link to a due diligence doc